### PR TITLE
update int8 flag for model conversion

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

File: llm_bench/python/convert.py [hunk 1]
```diff
@@ -1354,10 +1354,10 @@ def main():
         "-c",
         "--compress_weights",
         type=str,
-        choices=["INT8", "4BIT_DEFAULT", "INT4_SYM", "INT4_ASYM"],
+        choices=["INT8", "INT8_ASYM" "4BIT_DEFAULT", "INT4_SYM", "INT4_ASYM"],
```
Comment (eaidova) [inline llm_bench/python/convert.py:1357]:
```suggestion
        choices=["INT8", "INT8_ASYM",  "4BIT_DEFAULT", "INT4_SYM", "INT4_ASYM"],
```

File: llm_bench/python/utils/nncf_utils.py [hunk 2]
```diff
@@ -16,6 +16,11 @@
 }
 
 
+if "INT8_ASYM" in nncf.CompressWeightsMode.__members__:
+    COMPRESSION_OPTIONS["INT8_ASYM"] = {"mode": nncf.CompressWeightsMode.INT8_ASYM}
+
```
Comment (eaidova) [inline llm_bench/python/utils/nncf_utils.py:21]:
```suggestion
```

File: llm_bench/python/convert.py [hunk 3]
```diff
@@ -1354,10 +1354,10 @@ def main():
         "-c",
         "--compress_weights",
         type=str,
-        choices=["INT8", "4BIT_DEFAULT", "INT4_SYM", "INT4_ASYM"],
+        choices=["INT8", "INT8_ASYM",  "4BIT_DEFAULT", "INT4_SYM", "INT4_ASYM"],
         nargs="+",
         help=(
-            "The weight compression option, e.g. INT8 - INT8 weights, "
+            "The weight compression option, e.g. INT8 - INT8 weights (deprecated, please use INT8_ASYM instead)"
```
Comment (eaidova) [inline llm_bench/python/convert.py:1360]:
```suggestion
            "The weight compression option, e.g. INT8 - INT8 weights (deprecated, please use INT8_ASYM instead), "
```

**Comment by peterchen-intel** [comment]
@eaidova  Can you please also add her? https://github.com/openvinotoolkit/openvino.genai/blob/a3168aaaecef2a12abc147934053907f17e32fb9/llm_bench/python/utils/model_utils.py#L223
OV_FP16-INT8_ASYM and OV_FP32-INT8_ASYM , also PT_FP16-INT8_ASYM and PT_FP32-INT8_ASYM

**Review by AlexKoff88 — APPROVED** [review]

**Pull request merged**